docs/design-language.md

```diff
@@ -223,4 +223,45 @@ Panels come with an assortment of configurable navigation controls:
 
 ![panels](/img/panels/panel-welcome.jpg "Welcome Dialog Panel") ![panels](/img/panels/panel-feature.jpg "Features Panel")
 
+## Dialogs
+
+### Modal Dialogs
+
+Occasionally, there will be a need to display information, warn the user, or present controls in a manner that needs to utilize a more focused interface. Modal dialogs provide a flexible layout strategy to allow the user to put any content they want before the user and draws their focus to it, setting the rest of the app into a non-interactive state until it is dismissed. This allows important information to be displayed, and preserves the background information.
+
+### Examples
+
+![modals](/img/modals/measurement-units.jpg "Units under the measurement toolbar")<br/> _Units under the measurement toolbar_
+
+![modals](/img/modals/polygon-styles.jpg "Polygon Styles under the draw menu")<br/>_Polygon Styles under the draw toolbar_
+
+![modals](/img/modals/share-my-map.jpg "Share my Map")<br/>_Share my Map_
+
+![modals](/img/modals/save-project.jpg "Save Project")
+![modals](/img/modals/load-project.jpg "Load Project")<br/>_Save & Load Project_
+
+Additional Information regarding modal alerts can be found in the **<a href="https://developers.geocortex.com/docs/web/tutorial-import-map-model/#create-a-custom-component-with-a-model" target="_blank">Developer Center Information</a>**.
+
+### Alerts
+
+Dialogs are a good method of presenting alerts to the user, since they allow information that is relevant to a state change, or problem, without navigating away, or impacting the information within the app. This can include, but are not limited to:
+
+-   Destructive actions that would otherwise destroy information that has been altered.
+
+![dialogs](/img/dialogs/dialog.png "Dialog")
+
+Alerts must always preface the problem, and offer the user the ability to dismiss, edit, or disregard the problem in question.
+
 ---
+
+### Notifications
+
+Problems that might appear with some actions, including saving, adding information or connection issues, need to display information to the user. These are typically not critical issues, and simply need to appear for the knowledge of the user. Notifications will often appear in a non-invasive manner, dismissing either automatically after a certain time period, or by the user.
+
+Notifications can either be presented with inline tips, or by using the toast notification system.
+
+![notifications](/img/dialogs/notification-toast.jpg "Toast Notification")
```

Comment:
Screenshots should be taken in Web and not from Designer.

Reply:
Reach out to a Web team Developer if you need help finding how to get various notifications to display.

Reply:
This has been done and the notifications changed.